comfyui-truss/README.md

```diff
@@ -0,0 +1,312 @@
+## ComfyUI Truss
+
+This truss is designed to allow ComfyUI users to easily convert their workflows into a production grade API service. 
+
+## Exporting the ComfyUI workflow
+
+Inside ComfyUI, you can save workflows as a JSON file. However, the regular JSON format the ComfyUI uses will not work. Instead, the workflow has to be saved in the API format. Here is how you can do that:
+
+1. Go to ComfyUI and click on the gear icon for the project
+![gear_icon](images/comfyui-screenshot-1.png)
+
+2. Next, checkmark the box which says `Enable Dev Mode Options`
+![enable_dev_mode_options](images/comfyui-screenshot-2.png)
+
+3. Now, if you go back to the project you will see an option called `Save(API Format)`. This is the one you want to use to save your workflow. Using this method you can save any ComfyUI workflow as a JSON file in the API format.
+![save_api_format](images/comfyui-screenshot-3.png)
+
```

Comment:
Consider moving this section down to right before invocation to make things flow better, or at least mention here that the file we're creating will be used during model invocation